src/resources/Pipelines/Statements/StatementsInterfaces.ts

```diff
@@ -0,0 +1,80 @@
+export enum StatementsFeature {
+    Filter = 'filter',
+    Trigger = 'trigger',
+    Ranking = 'ranking',
+    Stop = 'stop',
+    Thesaurus = 'thesaurus',
+    Top = 'top',
+    TopClicks = 'topClicks',
+    QuerySuggest = 'querySuggest',
+    RankingWeight = 'rankingweight',
+    Recommendation = 'recommendation',
+    QueryParamOverride = 'queryParamOverride',
+}
```

Comment:
I'm not sure, but you might have to move the enums into file https://github.com/coveo/platform-client/blob/81abffc5c623541e12bb187b58e687aaf6309ec3/src/resources/Enums.ts, otherwise typescript won't be including them in the .js which il result in runtime errors.

Reply:
You are right